i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-keysight-nvos-restapi.md

```diff
@@ -199,103 +191,13 @@ apt install centreon-plugin-network-keysight-nvos-restapi
 3. Appliquez le modèle d'hôte **Net-Keysight-Nvos-Restapi-custom**.
 4. Une fois le modèle appliqué, les macros ci-dessous indiquées comme requises (**Obligatoire**) doivent être renseignées.
 
-| Obligatoire | Macro                       | Description                                                                       | Défaut  |
-|:------------|:----------------------------|:----------------------------------------------------------------------------------|:--------|
-| x           | KEYSIGHTNVOSAPIPASSWORD     | API password                                                                      |         |
-|             | KEYSIGHTNVOSAPIPORT         | Port used                                                                         | 8000    |
-|             | KEYSIGHTNVOSAPIPROTO        | Specify https if needed                                                           | https   |
-| x           | KEYSIGHTNVOSAPIUSERNAME     | API username                                                                      |         |
-|             | KEYSIGHTNVOSAPIEXTRAOPTIONS | Any extra option you may want to add to every command line (eg. a --verbose flag) |         |
-
-
-### Service 
```

Comment:
Why are we losing the "Service" section?

Reply:
because it was done with the script version before